deployment/helm/node-feature-discovery/values.yaml

```diff
@@ -395,7 +395,6 @@ topologyUpdater:
   rbac:
     create: false
 
-  kubeletConfigPath:
```

Comment:
Again, I don't think we should touch this(?)

Another note: do we want to add a config field for the new kubelet state dir command line flag?

Reply:
I revert the removal of this value and added the missed kubelet state dir flag. Thanks!